spec/controllers/policies_controller_spec.rb

```diff
@@ -95,7 +95,7 @@ def object_for_index
     describe "index" do
       routes { Mongoidable::Engine.routes }
       it "returns user type policies" do
-        database_policies = 10.times do |index|
+        database_policies = Array.new(10) do |index|
```

Comment:
Why did you make this change?

Reply:
`10.times` returns the count of the number of times it ran (10).  `Array.new(10)` returns the new array (size 10) with the new values.

We want the array of values, not the count.  This test was not working previously.